src/DotNetCore.CAP/Messages/Headers.cs

```diff
@@ -28,5 +28,10 @@ public static class Headers
         public const string SentTime = "cap-senttime";
 
         public const string Exception = "cap-exception";
+        
+        /// <summary>
+        /// Specifies a destination Topic where the Message will be sent to, if there are Multiple topics configured.
+        /// </summary>
+        public const string Destination = "cap-destination";
```

Comment:
Do we need to define this header into `AzureServiceBusHeaders`? Currently it seems that it only used in ASB?

Reply:
You're right @yang-xiaodong ! Moved this to AzureServiceBusHeaders instead.